tests/test_request.py

```diff
@@ -5,63 +5,63 @@
 
 
 @pytest.fixture
-def request():
+def test_request():
```

Comment:
I think we should avoid naming fixtures with a `test` prefix since that's how tests are named.
```suggestion
def sample_request():
```

Reply:
done